pyproject.toml

```diff
@@ -0,0 +1,67 @@
+[project]
+name = "openvino"
+dynamic = ["version", "entry-points", "scripts"]
+description = "OpenVINO(TM) Runtime"
+readme = {file = "README.md", content-type="text/markdown"}
+license = {text = "OSI Approved :: Apache Software License"}
+authors = [
+    { name = "OpenVINO Developers", email = "openvino@intel.com" },
+]
+classifiers = [
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+]
+requires-python = ">=3.9"
+
+dependencies = [
+    "numpy>=1.16.6,<2.2.0",
+    "openvino-telemetry>=2023.2.1",
+    "packaging"
+]
+
+[build-system]
+requires = ["setuptools>=70,<=75", "wheel<=0.45.1", "cmake<=3.31.6"]
```

Comment:
do we still need https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/wheel/requirements-dev.txt ?

Reply:
Also I don't see in that requirements file that `cmake` is required for building python api wheels